### DISCUSSION TITLE
a problem running the cartpole/config/parallel /cartpole_dqn_config.py

### DISCUSSION THREAD

**Discussion by hulingxiang666** [opendilab/DI-engine#719] [Q&A] [answered]
I'm a beginner of DI-engine. When I was running the the file cartpole/config/parallel /cartpole_dqn_config.py. The program didn't have any output,the CPU usage was also shown as 0,even I couldn't interrupt kernel.

**Comment by PaParaZz1** [comment, marked as answer]
For the parallel demo, you can refer to our new task pipeline design. ([link](https://github.com/opendilab/DI-engine/blob/main/ding/example/dqn.py#L12))
